src/app/courses/add-courses/courses-add.component.html

```diff
@@ -1,102 +1,109 @@
-<h1 i18n>Add Courses</h1>
-<form [formGroup]="courseForm" (ngSubmit)="onSubmit()" novalidate>
-  <div>
-    <mat-form-field>
-      <input matInput i18n-placeholder placeholder="Course Title" formControlName="courseTitle" required>
-      <mat-error><planet-form-error-messages class="km-coursetitle-errormessage" [control]="courseForm.controls.courseTitle"></planet-form-error-messages></mat-error>
-    </mat-form-field>
-    <mat-form-field>
-      <input matInput i18n-placeholder placeholder="Language of Instruction" formControlName="languageOfInstruction">
-    </mat-form-field>
-    <mat-form-field>
-      <input matInput i18n-placeholder placeholder="Member Limit" type="number" formControlName="memberLimit">
-      <mat-error><planet-form-error-messages [control]="courseForm.controls.memberLimit"></planet-form-error-messages></mat-error>
-    </mat-form-field>
-  </div>
+<mat-toolbar>
+  <a mat-icon-button routerLink="/courses"><mat-icon>arrow_back</mat-icon></a>
+</mat-toolbar>
 
-  <div>
-    <mat-form-field>
-      <mat-select i18n-placeholder placeholder="Course Leader" formControlName="courseLeader">
-        <mat-option *ngFor="let memb of members" value={{memb?.id}}> {{memb?.doc.firstName}} {{memb?.doc.middleName}} {{memb?.doc.lastName}}</mat-option>
-      </mat-select>
-    </mat-form-field>
-    <mat-form-field>
-      <input matInput i18n-placeholder placeholder="Method" formControlName="method">
-    </mat-form-field>
-  </div>
+<div class="space-container">
+  <mat-toolbar class="primary-color font-size-1" i18n>Add new Course</mat-toolbar>
```

Comment:
it has changed the title can we use the same title or atleast all words to have same case sensitivity.